### DISCUSSION TITLE
Hi kiali community

### DISCUSSION THREAD

**Discussion by Xunzhuo** [kiali/kiali#4639] [General]
Hi community,

I am Xunzhuo from China, and I am recently working on putting kiali into large scale production in Tencent Cloud Mesh.

I believe that you folks can notice that I am quite active in Kiali in Slack, issue, discussion, PRs (BTW, I am sorry for some minor fix PRs, which may take you forks some time to review).

I accept suggestions from @lucasponce and @jmazzitelli , minor fix PRs won't help to kiali, but takes extra time from reviews of the maintainers.

So I open this discussion to discuss changes that I plan to make before giving PRs in the future. 

And also I will ask maintainers to assign me to some issues that I am interested in, to help to implement some features PRs.

Hope that I can help more to kiali and help to grow kiali together. :)

**Comment by lucasponce** [comment]
Welcome, @Xunzhuo, and thanks again for your contributions.
If you'd like to contribute to the project, there are some issues with "good first issue" that may be a good start:
https://github.com/kiali/kiali/issues/4276

The purpose would be to get familiar with the structure of the project and at the same time to work in some feature that is needed.

Also, there are some features marked with "help wanted" but perhaps they may show more complexity for a newcomer:

https://github.com/kiali/kiali/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22

We haven't reviewed if other issues can also be a "good first issue" so feel free to take a look and don't hesitate to express interest in some issues; the project leads (@jshaughn @abonas or @lucasponce) may help you to be on board in the coding. Also, they can coordinate if some issue is already being part of any work in progress by another colleague.

Thanks !

**Comment by Xunzhuo** [comment]
Thanks @lucasponce. I am interested in https://github.com/kiali/kiali/issues/4604, if it is OK, please assign it to me.

**Comment by Xunzhuo** [comment]
Hi @lucasponce , there is a fact that I want you know is that I am not an expert in frontend(ui), so maybe I will not help more in kiali ui, but any tasks in backend (kiali/kiali, kiali/kiali-operator, kiali/helm-charts) I can all help with.

**Reply by jshaughn** [reply]
#4604 would involve some server-side support to probably perform a compatibility check.  The UI-side would be fairly limited, I think.

**Comment by Xunzhuo** [comment]
## Contributions:
> Record some PRs and Issues of mine here.
### PRs of mine:
#### kiali/demos:
https://github.com/kiali/demos/pull/46
https://github.com/kiali/demos/pull/43
#### kiali/kiali:
https://github.com/kiali/kiali/pull/4728
https://github.com/kiali/kiali/pull/4722
https://github.com/kiali/kiali/pull/4690
https://github.com/kiali/kiali/pull/4689
https://github.com/kiali/kiali/pull/4646
https://github.com/kiali/kiali/pull/4633
https://github.com/kiali/kiali/pull/4630
https://github.com/kiali/kiali/pull/4629
https://github.com/kiali/kiali/pull/4628
https://github.com/kiali/kiali/pull/4620
https://github.com/kiali/kiali/pull/4616
https://github.com/kiali/kiali/pull/4614
https://github.com/kiali/kiali/pull/4602
#### kiali/kiali-ui:
https://github.com/kiali/kiali-ui/pull/2302
#### kiali/kiali.io:
https://github.com/kiali/kiali.io/pull/509
https://github.com/kiali/kiali.io/pull/498
#### kiali/kiali-operator:
https://github.com/kiali/kiali-operator/pull/472
#### kiali/helm-charts:
https://github.com/kiali/helm-charts/pull/121

### Issues of mine:
https://github.com/kiali/kiali/issues/4638
https://github.com/kiali/kiali/issues/4664
https://github.com/kiali/kiali/issues/4649

### Discussion I started in Github Discussion or Slack:
........
